tests/playwright/pages/editor-page.js

```diff
@@ -208,20 +208,17 @@ module.exports = class EditorPage extends BasePage {
 	 *
```

Comment:
@davseve What are these changes about in `tests/playwright/pages/editor-page.js`?

Reply:
The `selectElement` Playwright function avoided selecting sticky elements by a condition.
For some reason, the nested accordion met the condition that the `selectElement` was avoiding.
For that reason, I talked to @hein-obox and he suggested an improved `selectElement` function that isn't relating elements that are not sticky.